src/pynxtools/dataconverter/nexus_tree.py

```diff
@@ -245,6 +245,9 @@ def search_child_with_name(
                 return self.add_inherited_node(name)
         return None
 
+    def get_children_for(self, xml_elem: ET._Element) -> Optional[ET._Element]:
+        return next((x for x in self.children if x.inheritance[0] == xml_elem), None)
```

Comment:
I guess this could benefit from a docstring.

Reply:
Ah yes, I forgot to write it. Thanks I'll update it

Reply:
Added